Tests/SentryTests/SentryCrash/SentryStacktraceBuilderTests.swift

```diff
@@ -68,6 +68,32 @@ class SentryStacktraceBuilderTests: XCTestCase {
         XCTAssertTrue(filteredFrames.count == 1, "The frames must be ordered from caller to callee, or oldest to youngest.")
     }
      
+    @available(macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0, *)
+    func testConcurrentStacktraces() async throws {
+        guard #available(macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0, *) else { return }
```

Comment:
I don't know why, but adding `@available(macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0, *)` dont stop the test from running. I had experience this before.
Need to check in the code.

Reply:
@available doesn't affect which tests are run, it's tripped me up in the past too. you have to do something like 
```
guard @available(...) else {
	throw XCTSkip("don't run this test for iOS < X")
}
```